libvcx/src/aries/handlers/issuance/holder/state_machine.rs

```diff
@@ -60,10 +60,10 @@ impl HolderSM {
 
         for (uid, message) in messages {
             match self.state {
-                HolderState::OfferReceived(_) => {
+                HolderFullState::OfferReceived(_) => {
```

Comment:
This code can look slightly cleaner if we match against `self.get_state()` instead

Reply:
That's true, I'll do that everywhere

Reply:
Yes, but then it would be inconsistent with other SMs where the enums holds useful data.